examples/particles.ipynb

```diff
@@ -149,17 +149,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from expertsystem.data import Particle\n",
+    "from expertsystem.state.particle import create_particle\n",
     "\n",
-    "new_omega = Particle(\n",
-    "    name=omega.name,\n",
-    "    pid=omega.pid,\n",
-    "    mass=omega.mass,\n",
-    "    width=0.01,\n",
-    "    state=omega.state,\n",
-    ")\n",
-    "particle.DATABASE.add(new_omega)\n",
-    "particle.DATABASE[\"omega(782)\"].width"
+    "new_N1650_minus = create_particle(N1650_minus, name=\"N(1650)-_modified\", width=0.2)\n",
+    "\n",
+    "particle.DATABASE.add(new_N1650_minus)\n",
+    "particle.DATABASE[\"N(1650)-_modified\"].width"
```

Comment:
I would remove the `.width`, so that Jupyter prints the complete new `Particle` instance. But up to you, it may also make the output cell look ugly